packages/identity/src/errors/__tests__/index.ts

```diff
@@ -1,6 +1,6 @@
-// @flow
-
 // require all modules ending in ".spec.js" or ".spec.web.js" from the
 // current directory and all subdirectories
-const testsContext = require.context('.', true, /\.spec(\.web)?\.js$/);
+// @ts-expect-error
```

Comment:
Can you add a description? Bonus points if you can point to an issue in typescript's issue tracker.